CHANGES.rst

```diff
@@ -2,8 +2,12 @@
  Release History
 =================
 
-2.0.1 (unreleased)
-==================
+3.0 (unreleased)
+================
```

Comment:
Bikeshedding topic: 3.0 or 3.0.0?

Reply:
I personally do not like version numbers containing two dots and ending in `0`. I think `3.0.0` contains no more information than `3.0`. `3.0.1` can be used if If there is a bugfix needed. Development does not have the primary goal of bugfixes, so the version number before `(unreleased)` should be the next minor or even major version.

On the other hand I see the problem for native english speakers to distinguish between the version number `3.0` and the decimal number `3.0`, `3.0.0` can only be a version number.

Reply:
Also, I think if you use 3.0, then zest.releaser will automatically suggest 3.1 as the next version, whereas if you use 3.0.0, zest.releaser will default to 3.0.1.

Reply:
That's why I prefer 3.0 to get 3.1 as the next version (to develop towards the next feature release instead of the next bug fix). This allows to use `fullrelease --no-input` which is a time saver.